test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -71,10 +71,10 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
 		$functionDefinition>$functionDefinitionFile
 
         $PowerShell = [powershell]::Create()
+        $PowerShell.AddScript(". $functionDefinitionFile").Invoke()

```

Comment:
there should be corresponding `$PowerShell.Commands.Clear()` to avoid re-executing it on the first run.
Also out of curiosity, did you measure the time before and after?